libzeth/circuits/joinsplit.tcc

```diff
@@ -295,81 +278,54 @@ public:
             // Initialize the unpacked input corresponding to the variables of
             // size different to 253 (FieldT::capacity()) bits (smaller inputs
             // and residual bits). We obtain an unpacked value equal to v_in ||
-            // v_out || h_sig || nf_{1..NumInputs} || cm_{1..NumOutputs} ||
-            // h_i_{1..NumInput}. We fill them here in reverse order because of
-            // the use of .insert() function which adds a variable before a
+            // v_out || h_sig || nf_{1..NumInputs} || h_i_{1..NumInput}.
+            // We fill them here in reverse order because of the use of
+            //  the function .insert() function which adds a variable before a
```

Comment:
` the function .insert() function which adds a variable before ` -> let's change "the function .insert" to " the .insert() function [...]"

Reply:
Updated.